### ISSUE TITLE
If I install on my own server?

### ISSUE THREAD

**Issue by coderam** [bitwarden/core#137]
With bitwarden.sh? Could you allow us to specify ports like 8080 instead of 80 and 9898 for SSL?  This would be great because many people would want to install this on their VPS where those ports may be used up?

Thanks!!

**Comment by kspearrin** [comment]
But you can...?

**Comment by sorcer1122** [comment]
I have the very same problem. I have got a server with nginx running on default ports 80 and 443, so I changed those ports during the installation of bitwarden to 7070 and 7071 correspondingly. The problem is nginx by default still uses port 80 and then redirect it to 7070, which is not going to work for me since my original nginx uses port 80.

So, my question is how to reassemble bitwarden/nginx image and to point it to ports 7070 and 7071?
